crates/net/network/src/cache.rs

```diff
@@ -1,5 +1,15 @@
+use derive_more::{Deref, DerefMut};
 use linked_hash_set::LinkedHashSet;
-use std::{borrow::Borrow, hash::Hash, num::NonZeroUsize};
+use schnellru::{self, ByLength};
```

Comment:
```suggestion
use schnellru::ByLength;
```

Reply:
I'm using `schnellru::LruMap` to be able to make new type with same name `LruMap`, so I need to bring the whole module into scope. I forgot to add schnellru crate to the manifest though when I moved this code out to its own branch yesterday that's why clippy says the external crate doesn't exist.